README.md

```diff
@@ -768,6 +769,90 @@ loggingClient.getEntries(function(err, entries) {
 ```
 
 
+## Google Cloud Natural Language (Beta)
+
+> **This is a Beta release of Google Cloud Natural Language.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-language-docs]
+- [Official Documentation][cloud-language-docs]
+
+#### Using the all-in-one module
+
+```
+$ npm install --save google-cloud
+```
+
+```js
+var gcloud = require('google-cloud');
+var language = gcloud.language;
+```
+
+#### Using the Natural Language API module
+
+```
+$ npm install --save @google-cloud/language
+```
+
+```js
+var language = require('@google-cloud/language');
+```
+
+#### Preview
+
+```js
+// Authenticating on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var languageClient = language({
+  projectId: 'grape-spaceship-123',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Get the entities from a sentence.
+languageClient.detectEntities('Stephen of Michigan!', function(err, entities) {
+  // entities = {
+  //   people: ['Stephen'],
+  //   places: ['Michigan']
+  // }
+});
+
+// Create a document if you plan to run multiple detections.
+var document = languageClient.document('Contributions welcome!');
+
+// Analyze the sentiment of the document.
+document.detectSentiment(function(err, positiveSentiment) {
+  // positiveSentiment = true
+});
+
+// Parse the syntax of the document.
+document.annotate(function(err, annotations) {
+  // annotations = {
+  //   language: 'en',
+  //   sentiment: true,
```

Comment:
According to the JSDoc for the `Document` object, `sentiment` should be a number, but here it is represented as a boolean.